### PR TITLE
RK-10432 - Adding Cloud native debug session section

### PR DESCRIPTION
**Pull request by noagoldman**

### CODE REVIEW

File: docs/debug-session-setup.md [hunk 1]
```diff
@@ -39,3 +39,9 @@ Click **Start Debugging** to start your debug session.
 
 Click on the purple arrow from your current debug session to **edit and change selected instances**.
 
+### Cloud-Native Debug Session
+Visualize your Kubernetes environment. Rookout’s Cloud Native Debug Session allows you to group and filter by Namespace, Cluster, and other Kubernetes-specific attributes, giving you a clear visual display of your application’s pods and clusters grouped by K8S-driven categories.
```
Comment (dan505512) [inline docs/debug-session-setup.md:43]: No cluster. It's only the namespace and deployment

Comment (noagoldman) [reply docs/debug-session-setup.md:43]: It's ok it's for PR purpses

**Review by dan505512 — APPROVED** [review]

**Review by lirook — APPROVED** [review]

**Pull request merged**